features/include/pcl/features/boundary.h

```diff
@@ -41,6 +41,7 @@
 #pragma once
 
 #include <pcl/features/feature.h>
+#include <pcl/point_types.h>       // for pcl::Vector4fMapConst
```

Comment:
If point_types.h is only needed for `Vector4fMapConst` here, we could replace `pcl::Vector4fMapConst` by `Eigen::Vector4f` instead of including point_types.h